code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Medicine.dm

```diff
@@ -650,3 +650,27 @@
 
 /datum/reagent/ipecac/affect_blood(var/mob/living/carbon/M, var/alien, var/removed)
 	M.adjustToxLoss(2 * removed) //If you inject it you're doing it wrong
+
+/datum/reagent/kelotanetopical
```

Comment:
...I didn't mean the type path (which is now broken).
I meant like, have the topical variant have its own name, or a name derived from "Kelotane" instead of just "Topical Kelotane"